lib/actions/map.js

```diff
@@ -47,11 +46,23 @@ export function clearLocation(payload) {
   }
 }
 
-export function zoomToStop(stop) {
+/**
+ * Centers the given map to the coordinates of the specified place.
+ */
+export function setMapCenter(map /* MapRef */, location, zoom) {
+  return function () {
+    const { lat, lon } = location
+    if (map && !isNaN(lat) && !isNaN(lon)) {
+      map.jumpTo({ center: [lon, lat], zoom })
+    }
+  }
+}
+
+export function zoomToStop(map /* MapRef */, stop, zoom) {
   return function (dispatch, getState) {
-    if (!stop) return
-    dispatch(setMapZoom({ zoom: 17 }))
-    dispatch(setMapCenter({ lat: stop.lat, lon: stop.lon }))
+    if (stop && map) {
+      map.jumpTo({ center: [stop.lon, stop.lat], zoom: zoom || 17 })
```

Comment:
This is not working for me in the stop viewer. I think it's conflicting with the `flyTo` that's also being fired by it.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/666/commits/747508b0dafb840ca6c0635c8d14ade09512a76e.